src/shared/stores/index.js

```diff
@@ -73,26 +72,20 @@ export default types
     },
   }))
   .actions(self => ({
-    getNextPage: flow(function*() {
+    getNextPage() {
       const { connection } = self.root;
 
-      const { type, id, page } = connection.selectedColumn.items[
-        connection.selectedColumn.items.length - 1
-      ];
+      const [{ type, id, page }] = connection.selectedColumn.items.slice(-1);
```

Comment:
Uhh... I had never seen a `.slice(-1)` to get the last item of an array! Nice trick.